src/main/scala/unchained/Height.scala

```diff
@@ -0,0 +1,50 @@
+package unchained
+
+import akka.NotUsed
+import akka.util.ByteString
+
+import akka.stream.scaladsl.{ Flow, Source }
+
+/**
+ * [[https://bitcoin.org/en/glossary/block-height Block height]]
+ *
+ * @param value unsigned integer representation
+ */
+class Height(val value: BigInt) extends AnyVal {
+  @inline override def toString = value.toString
+}
+
+object Height {
+  /**
+   * Returns the [[Height]] representation for the given numeric value.
+   */
+  def apply[N: Numeric](numeric: N): Height =
+    new Height(BigInt(implicitly[Numeric[N]] toLong numeric))
+
+  /**
+   * Block [[Height]] parser from the given binary representation.
+   */
+  def parser: Flow[ByteString, Height, NotUsed] =
```

Comment:
How do you plan to use this ? when in the code will it be used ?

Reply:
First in Coinbase parser, but this type seems common